### PR TITLE
Enable not GCed leak tracking.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

**Review by goderbauer — APPROVED** [review]
LGTM

File: packages/flutter/test/foundation/leak_tracking.dart [hunk 1]
```diff
@@ -35,6 +35,9 @@ class WeakSet {
   }
 }
 
+/// If true, the leak tracker will collect debug information for leaks.
+bool collectDebugInformationForLeaks = false;
```
Comment (goderbauer) [inline packages/flutter/test/foundation/leak_tracking.dart:39]: Is this a debug only flag? Then it should be named something like "debugFoo". It should also be moved to debug.dart. 

And the documentation should be extended.

**Review by goderbauer — CHANGES_REQUESTED** [review]

File: packages/flutter/test/foundation/leak_tracking.dart (hunk 1 above)
Comment (polina-c) [reply packages/flutter/test/foundation/leak_tracking.dart]: Removed it for now as it turned out I did not use it for debugging.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**